sdk/tests/conformance2/reading/read-pixels-pack-parameters.html

```diff
@@ -35,9 +35,18 @@
 </head>
 <body>
 <canvas id="example" width="4" height="4"></canvas>
+<canvas id="canvas2d" width="4" height="4"></canvas>
 <div id="description"></div>
 <div id="console"></div>
 <script>
+"use strict"
+
+var wtu = WebGLTestUtils;
+var expectedColor = [[255, 102, 0, 255],
+                     [2, 200, 102, 255],
+                     [134, 87, 234, 255],
+                     [99, 5, 76, 255]];
```

Comment:
The reason I don't use the standard colors is that the bugs are likely 1 or more bytes shifted due to incorrect computation.  So with random numbers instead 255 and 0, it's easier to understand what the bug could be.